lib/yarp.rb

```diff
@@ -234,5 +234,5 @@ def self.load(source, serialized)
 require_relative "yarp/node"
 require_relative "yarp/ripper_compat"
 require_relative "yarp/serialize"
-require_relative "yarp/yarp"
 require_relative "yarp/pack"
+require "yarp.so"
```

Comment:
Is this extension correct on all platforms? (Honestly asking I don't know)

Reply:
As far as I understand, yes!